.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: test
+on:
+- pull_request
+jobs:
+  imperial_macos:
+    runs-on: macos-latest
+    env:
+      DEVELOPER_DIR: /Applications/Xcode_11.4_beta.app/Contents/Developer
+    steps:
+    - uses: actions/checkout@v2
+    - run: brew install pkg-config libressl
```

Comment:
libressl is not required for Vapor 4

Reply:
See my comments over on the Vapor PR re: this